mocks/CategoriesManager/categories.js

```diff
@@ -0,0 +1,56 @@
+const faker = require('faker');
```

Comment:
This is file can be removed, no such api endpoint. You can aggregate data from several api